### ISSUE TITLE
NCLT datasets PR GT

### ISSUE THREAD

**Issue by ydragon719** [SCNU-RISLAB/OverlapMamba#1]
Your work is fantastic! I'm wondering if the file named "nclt_0108_0205.npy" is the ground truth calculated using 0205 as the query and 0108 as the database through the overlap method? Could you provide other ground truth files used in the paper's experiments, such as "nclt_0108_0615.npy" and "nclt_0108_0405.npy", for testing purposes? Looking forward to your reply!

**Comment by JiehaoLuo** [comment]
Yes, you are correct. The file "nclt_0108_0205.npy" is indeed the ground truth calculated using 0205 as query and 0108 as database through the overlap method. The NCLT ground truth generation process is as followed:

1. Generate poses using [pole-localization](https://github.com/PRBonn/pole-localization) repository's [pynclt.py](https://github.com/PRBonn/pole-localization/blob/main/src/pynclt.py) script. This requires .bin files arranged according to repository requirements and produces pose files named `T_w_r_gt_velo_DATE.npy`.
2. Generate ground truth using [gen_ground_truth.py](https://github.com/BIT-MJY/SeqOT/blob/main/data_prepararion/gen_ground_truth.py) script from [SeqOT](https://github.com/BIT-MJY/SeqOT). This script takes two pose files (database and query) and determines distances/overlaps between points, outputting files named `nclt_DATABASE_QUERY.npy`.

Regarding the ground truth files you requested ("nclt_0108_0615.npy" and "nclt_0108_0405.npy"), these will be released after our paper is accepted!
